source/includes/steps-install-shell-macos.yaml

```diff
@@ -4,30 +4,48 @@ source:
   ref: navigate-dlc
 ref: navigate-dlc-macos
 ---
-source:
-  file: steps-install-shell-base.yaml
-  ref: download-archive
+title: "Download the ``mongosh`` installation archive for your
+  operating system."
 ref: download-archive-macos
+level: 4
+content: |
+   Download the appropriate version of ``mongosh`` for your operating
+   system. MongoDB provides versions of ``mongosh`` for Intel and ARM 
+   architectures. 
+   
+   See the `MongoDB Download Center
+   <https://www.mongodb.com/try/download/shell?jmp=docs>`__.
+
 ---
 title: "Extract the files from the downloaded archive."
 ref: extract-zip-macos
 level: 4
 content: |
-
-   Run the following command from the directory containing the
-   ``mongosh`` ``.zip`` archive:
+   Go to the directory that contains the ``mongosh`` ``.zip`` archive,
+   then unpack the ``.zip`` file. 
+   
+   If your computer is Intel based, run:
    
    .. code-block:: sh
 
       unzip mongosh-{+version+}-darwin-x64.zip
 
-   The extracted ``bin`` folder contains two binaries: ``mongosh`` and
-   ``mongosh_csfle_v1.dylib``.
+   If your computer is ARM based (M1 or M2), run:
+   
+   .. code-block:: sh
+
+      unzip mongosh-{+version+}-darwin-arm64.zip
+
+
+   The extracted archive has a ``bin`` folder that contains two files,
+   ``mongosh`` and ``mongosh_crypt_v1.dylib``.
 
    If your web browser automatically extracts the archive as part of the
-   download or you extract the archive without the ``unzip`` command,
-   you may need to make the binary executable. Run the following
-   command from the directory where you extracted the archive:
```

Comment:
(Some suggestions)

- "make" here is a bit ambiguous. Are you enabling execution? Or are you building the the binary? Consider switching up the wording for slight clarification. (Though it is obvious in context.)
- The whole piece about making the binary executable is really an optional additional step. What do you think about adding it as an optional step instead?